### PR TITLE
CMS-Bookstore - Milestone 3

### PR DESCRIPTION
**Pull request by udberg**
In this milestone, l connected the CREATE_BOOK and REMOVE_BOOK
actions with the React component to make the pages dynamic.

### CODE REVIEW

File: src/containers/BookList.js [hunk 1]
```diff
@@ -2,14 +2,15 @@ import React from 'react';
...
     <table>
       <thead>
         <tr>
           <th>Book ID</th>
           <th>Title</th>
           <th>Category</th>
+          <th>Remove Book</th>
         </tr>
       </thead>
       <tbody>
```
Comment (rindrajosia) [inline src/containers/BookList.js:16]: [REQUIRED] BookList component should passe handleRemoveBook to Book component

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- BookForm stores the title and the category in the state :+1: 
- BookForm implement `handleChange` and attaches it to input's and select's `onChange` event :100: 
- BookForm implements `handleSubmit` and attaches it to form's `onSubmit` event :+1: 
- `Book` component adds `Remove book` button to the book row :100: 
- `Book` component attaches passed `handleRemoveBook` to on click event of `Remove book` button :100: 

To improve :factory_worker: [REQUIRED]:

- [x]  [REQUIRED]  Please check the live demo link and screenshot in your readme because they are broken. Keep in mind that it is the first thing users see.
- [x] [REQUIRED] I cannot see any book when I open your APP in my browser. In fact, there should be a default list of books when I open your APP.
- [x]  [REQUIRED] I cannot add any book via your book form.
- [x]  [REQUIRED] `BookList` component should connect the `REMOVE_BOOK` action
- [x]  [REQUIRED] `BookList` component should passe `handleRemoveBook` to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)

Nice to have [OPTIONAL] :

- [x] event handlers inside `class` components use arrow functions to avoid explicit binding on `constructor`. That makes the code shorter, more reusable, and uses modern ES6 JavaScript.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi Udberg :slightly_smiling_face: 


* [x] The link pointing to your live demo isn't working.

However, I was able to interact with your application locally on my machine. I discovered that:
* [x]  The removeBook feature isn't working. Going through the implementation, I discovered you are passing the wrong prop **handleRemoveBook instead of removeBook** to your bookList component.

Revise your code, make the necessary changes to your readme and removeBook feature, then submit another code for a code review. 

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Review by christianotieno — APPROVED** [review]
Hi @udberg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**